lib/Cache/Core/CoreCacheHandler.php

```diff
@@ -543,7 +543,6 @@ function ($currentValue) {
         $item->set($data);
         $item->expiresAfter($lifetime);
         $item->tag($tags);
-        $item->tag($key);
```

Comment:
BC Break? If someone is relying on this tag?

Reply:
Why should someone do this? It can simply loaded/removed without this tag...

Reply:
Should we add a config to enable/disable this?